gcp_variant_transforms/libs/annotation/vep/vep_runner.py

```diff
@@ -203,20 +206,22 @@ def _process_pipeline_args(self, pipeline_args):
     # TODO(bahsir2): Fix the error messages of _check_flag since
     # --worker_machine_type has dest='machine_type'.
     self._machine_type = self._get_flag(flags_dict, 'machine_type')
-    # TODO(bashir2): Fall back to num_workers if max_num_workers is not set.
-    self._max_num_workers = self._get_flag(flags_dict, 'max_num_workers')
+    self._max_num_workers = self._get_flag(
+        flags_dict, 'max_num_workers', 'num_workers')
     if self._max_num_workers <= 0:
       raise ValueError(
-          '--max_num_workers should be a positive number, got: {}'.format(
-              self._max_num_workers))
+          '--max_num_workers and --num_workers should be positive numbers, '
+          'got: {}'.format(self._max_num_workers))
 
-  def _get_flag(self, pipeline_flags, flag):
+  def _get_flag(self, pipeline_flags, *expected_flags):
```

Comment:
nit: any reason for not using a list of strings instead of arbitrary num args?

Reply:
Slightly nicer format when calling this with just one flag (which is in most cases). I changed it to a list per your comment.

As a side: I am not sure how much all these list construction/slicing that we do impact performance of a Python code. I have been doing it fairly liberally but not sure if that is good or not. In other languages like C++ or Java, I would have thought more about it specially because the language's syntax does not make these as easy as Python usually does and performance in general is more centeric than Python. Thoughts?

Reply:
Okay, changed it back then :-)
And re. performance, yes definitely here is not a concern, I started thinking more about it when you told me about the '::' operator, but even there the performance concern was not significant compared to the much nicer code. I thought I bring it up as a general note to think about in coding/reviews.